.github/workflows/integration-test-tvOS16_1.yaml

```diff
@@ -1,4 +1,4 @@
-name: "Integration Test: tvOS 14.3"
+name: "Integration Test: tvOS 16.1"
```

Comment:
We need to update the Continuous Integration Testing of the README with this new version.

Reply:
https://github.com/ably/ably-cocoa/pull/1543/commits/2812a5e55478bb324d280c9b9db49dfd09f6ba59